reference/special-variables.markdown

```diff
@@ -33,6 +33,8 @@ Variables defined in a monitoring context.
 * [sys][sys]
 Variables defined in order to automate discovery of system values.
 
+* [def][def]
+Variables with some default value that can be overridden by [augments file][Augments].
```

Comment:
s/overridden/defined/

If you define something in the augments file, and you do not have special treatment in `bundle common def` it will be defined. But if you define a variable in `bundle common def` without a guard like `isvaribale()` then the variable as defined in policy wins.